AFQ/mask.py

```diff
@@ -238,12 +238,23 @@ def find_path(self, bids_layout, subject, session):
         pass
 
     def get_mask(self, api, row):
```

Comment:
This one doesn't have a docstring. What kind of thing is the `api` argument here?

Reply:
Is this an api.AFQ class instance? If so, we might want to call this input something else. Maybe `afq_object`?